backend/accounting-service/src/main/kotlin/services/Products.kt

```diff
@@ -219,7 +222,7 @@ class ProductService(
                     {
                         setParameter("provider", provider)
                     },
-                    "select * from products where provider = :provider order by priority, id"
+                    "select * from products where provider = :provider and hidden_in_grant_applications is false order by priority, id"
```

Comment:
This almost certainly breaks parts of the backend. If not now it will later. This is supposed to be hidden in grant applications, not hidden everywhere.